client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -59,6 +59,7 @@ const Checkout = React.createClass( {
 
 	componentWillMount: function() {
 		upgradesActions.resetTransaction();
+		analytics.luckyOrange.initialize();
 
```

Comment:
Does this mean lucky orange receives user data from calypso from the moment this particular component is mounted?